charts/juicefs/templates/fuse/daemonset.yaml

```diff
@@ -127,8 +127,9 @@ spec:
           - containerPort: 9567
             name: metrics
             protocol: TCP
-          {{- if .Values.fuse.privileged }}
           securityContext:
+            runAsUser: 0
```

Comment:
Will you consider about `configurable nonRoot`  like https://github.com/fluid-cloudnative/fluid/blob/master/docs/zh/samples/nonroot_access.md ?

```
runAs:
    uid: 1201
    gid: 1201
    user: fluid-user-1
    group: fluid-user-1
```

Reply:
It's different from Alluxio, JuiceFS needs FUSE and only root user can access FUSE. NonRoot users can access data in JuiceFS by setting `runAs` in application pod.

Reply:
Got it. thanks.